examples/redux/src/configuration.js

```diff
@@ -2,30 +2,30 @@ const configuration = {
   isEnabled: true,
   configurations: [
     {
-      origin: 'http://localhost:3000',
+      origin: "http://localhost:3006",
```

Comment:
```suggestion
      origin: "http://localhost:3000",
```

Reply:
pourquoi 3006 ?

Reply:
Because the app "example" runs on  port 3006. "start": "set PORT=3006&&react-scripts start".
So in order to take account of  authentication, i set port to 3006.